app/_locales/en/messages.json

```diff
@@ -2158,6 +2158,9 @@
   "tokenContractAddress": {
     "message": "Token Contract Address"
   },
+  "tokenDecimalFetchFailed": {
+    "message": "Token decimals not found."
```

Comment:
@NiranjanaBinoy I think we may need to explain this a bit more clearly to lay users?